src/Tutor.Core/DomainModel/KnowledgeComponents/KnowledgeComponentMastery.cs

```diff
@@ -54,6 +55,21 @@ public Result<AssessmentEvent> SelectSuitableAssessmentEvent(IAssessmentEventSel
             return assessmentEventSelector.SelectSuitableAssessmentEvent(KnowledgeComponent.Id, LearnerId);
         }
 
+        public Result SeekHelpForAssessmentEvent(int assessmentEventId, string helpType)
+        {
+            var assessmentEvent = KnowledgeComponent.GetAssessmentEvent(assessmentEventId);
+            if (assessmentEvent == null)
+                return Result.Fail("No assessment event with ID: " + assessmentEventId);
+
```

Comment:
I would like to check if the help type is appropriate for the type of AE here, but I'm not really sure what a good way to do that would be (or even what a good way to model help type would be, since the string I used for now obviously isn't really good).